daffodil-core/src/test/scala/org/apache/daffodil/dsom/TestRefMap.scala

```diff
@@ -247,10 +246,10 @@ class TestRefMap extends Logging {
         </xs:sequence>
       </xs:complexType>
       <xs:group name="g">
-        <xs:choice>
+        <xs:sequence><!-- a choice here would be UPA ambiguous -->
           <xs:element ref="ex:r1"/>
           <xs:element ref="ex:r1"/>
-        </xs:choice>
+        </xs:sequence>
```

Comment:
Is this changing the test? Or was this just a broken test that we weren't detecting correctly?

Reply:
Broken test. Was a choice between two identical element refs. That's a UPA error. Simplest kind actually. Validation detected this now, so had to fix it.